src/faketypes.jl

```diff
@@ -120,3 +123,47 @@ Base.:(==)(a::FakeTypeVar, b::FakeTypeVar) = a.lb == b.lb && a.name == b.name &&
 Base.:(==)(a::FakeUnionAll, b::FakeUnionAll) = a.var == b.var && a.body == b.body
 Base.:(==)(a::FakeUnion, b::FakeUnion) = a.a == b.a && a.b == b.b
 Base.:(==)(a::FakeTypeofBottom, b::FakeTypeofBottom) = true
+
+@static if !(Vararg isa Type)
+    struct FakeTypeofVararg
+        T
+        N
+        FakeTypeofVararg() = new()
+        FakeTypeofVararg(T) = new(T)
+        FakeTypeofVararg(T, N) = new(T, N)
+    end
+    function FakeTypeofVararg(va::Core.TypeofVararg)
+        if isdefined(va, :N)
+            FakeTypeofVararg(FakeTypeName(va.T), va.N)
+        elseif isdefined(va, :T)
+            FakeTypeofVararg(FakeTypeName(va.T))
```

Comment:
Is this more correct?
```suggestion
            FakeTypeofVararg(FakeTypeName(va.T; justname=true), va.N)
        elseif isdefined(va, :T)
            FakeTypeofVararg(FakeTypeName(va.T; justname=true))
```